src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { memo } from 'react';
+import cN from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+  onDelete: (todoId: number) => void
+  isLoading?: boolean
+};
+
+export const TodoItem: React.FC<Props> = memo(
+  ({
+    todo,
+    onDelete,
+    isLoading = true,
```

Comment:
Are you sure the default value should be `true`?

Reply:
@silchyn Yes, because I don't pass isLoading only to tempTodo, which is loading when waiting for adding of todo